packages/types/types/index.d.ts

```diff
@@ -193,18 +194,73 @@ export interface ThemeProviderProps {
   customBreakpoints?: string[]
 }
 
+export interface DividerProps
+  extends ColorProps,
+    SpaceProps,
+    WidthProps,
+    BorderColorProps {}
+
+export interface AvatarProps extends ClassNameProps, TextColorProps {
+  /** Heading displayed with avatar */
+  title?: string,
+  /** Subheading displayed beneath heading */
+  subtitle?: string,
+  src?: string,
+  /** Alternative initials to display when src is not provided */
+  initials?: string,
+  /** Size in px to use for height, width, and background-size */
+  size?: number
+}
+
+export interface BannerProps extends BoxProps {
+  header?: string | React.ComponentElement<IconProps, any>,
+  icon?: React.ComponentElement<IconProps, any>,
+  onClose?: (any) => void,
+  showIcon?: boolean,
+  text?: string | React.ComponentElement<any, any>,
+}
+
+export interface StampProps
+  extends SpaceProps, FontSizeProps, ColorProps {
+    variation?: 'outline' | 'fill' | 'solid',
+    size?: 'small' | 'medium'
+  }
+
+export interface ContainerProps {
+  maxWidth?: number
+}
+
+export interface FlagProps extends FlexProps,
+    ColorProps,
+    WidthProps {}
+
+export interface HugProps extends CardProps, ColorProps {
+  iconDisplay?: string[],
+  icon?: React.ComponentElement<IconProps, any>,
+  text?: React.ComponentElement<any, any> | React.ComponentElement<any, any>[] | string
```

Comment:
I think you probably want `ReactNode | string` here

Reply:
Thank you, wasn't quite sure what to use.

Reply:
I included the array since the PropTypes for that component has
```
  text: PropTypes.oneOfType([
    PropTypes.node,
    PropTypes.arrayOf(PropTypes.node),
    PropTypes.string,
  ]),
```

Reply:
Good call 👍 I've been referring to the React types when I'm not sure if something exists or not. Here's what I found for `ReactNode`.

```
    interface ReactNodeArray extends Array<ReactNode> {}
    type ReactFragment = {} | ReactNodeArray;
    type ReactNode = ReactChild | ReactFragment | ReactPortal | boolean | null | undefined;
```

Reply:
Ahhhh that's good to know! I'll update!